warehouse/query-core/src/test/java/datawave/query/CompositeFunctionsTest.java

```diff
@@ -670,6 +671,28 @@ public void testTLDWithLuceneAndIdentifierToLiteralEQJexl() throws Exception {
         }
     }
 
+
+    @Test
+    public void testWithHoles(){
+        eventQueryLogic.setQueryPlanner(new DatePartitionedQueryPlanner());
+        Map<String,String> extraParameters = new HashMap<>();
+        extraParameters.put("include.grouping.context", "true");
+        //TODO: why is HOLE not being pushed down with _Hole_ marker?!
```

Comment:
Is this still an issue?

Reply:
nope.  corresponding MR: https://github.com/NationalSecurityAgency/datawave/pull/2759

I know you like MRs, so I put some MRs in your MRs.